java-client/src/test/java/org/opensearch/client/opensearch/core/DeleteByQueryRequestTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
```

Comment:
Sorry if I wasn't clear. The correct header is as follows:

```
/*
 * SPDX-License-Identifier: Apache-2.0
 *
 * The OpenSearch Contributors require contributions made to
 * this file be licensed under the Apache-2.0 license or a
 * compatible open source license.
 */
```

Just copy it from other files.

Reply:
Done.